code/style.css

```diff
@@ -1,3 +1,227 @@
-body {
-  background: #FFECE9;
-}
\ No newline at end of file
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+  color: #3a3950;
+}
+
+/* Profile information: */
+.profile {
+  background-color: pink;
+  margin: auto;
+  display: block;
+  padding-top: 20px;
+}
+.user-img {
+  background-color: white;
+  display: block;
+  width: 70vw;
+  border-radius: 50%;
+  border: 5px solid #dddcf1;
+  padding: 5px;
+  margin-right: auto;
+  margin-left: auto;
+  margin-bottom: 20px;
+}
+.profile-info {
+  padding-bottom: 40px;
+}
+h2 {
+  color: rgb(255, 87, 87);
+  text-align: center;
+}
+.profile p {
+  text-align: center;
+}
+
+/* Big image section */
+.img-bag {
+  width: 100vw;
+  display: block;
+  padding: 20px 0;
+  margin: auto;
+  background-color: rgb(241, 204, 161);
+}
+
+/* My happy wave: */
+.wave {
+  height: 20px;
+  background-position: 0;
+  background-size: auto 100%;
+  background-repeat: repeat-x;
+  animation: animateCloud 20s linear infinite;
```

Comment:
Such a cool animation! Where did you find the wave-picture?

Reply:
Canva again :)